services/QuillLMS/app/assets/stylesheets/pages/progress_reports/class_report.scss

```diff
@@ -1,8 +1,68 @@
 .class-report {
-  .data-table {
+  max-width: 100vw;
+  #student-groupings-wrapper {
```

Comment:
Just moved this code over from `diagnostic_reports.scss`, because it's specific to this page.